OutOfSchool/OutOfSchool.ElasticsearchData/ESCompetitiveEventProvider.cs

```diff
@@ -31,6 +33,43 @@ public override async Task<SearchResultES<CompetitiveEventES>> Search(Competitiv
         };
     }
 
+    public override async Task<PriceRangeES> GetPriceRangeAsync(CompetitiveEventFilterES filter = null)
+    {
+        filter ??= new CompetitiveEventFilterES();
+
+        var query = CreateQueryFromFilter(filter);
```

Comment:
here might be the same case as with database, price may be included in query filter so need a way (parameter) to not add it so you can reuse this query builder.